geoalchemy2/admin/dialects/mysql.py

```diff
@@ -29,15 +29,16 @@ def reflect_geometry_column(inspector, table, column_info):
         return
 
     column_name = column_info.get("name")
+    schema = table.schema or inspector.default_schema_name
```

Comment:
What happens if `table.schema is None`? Is it allowed by MySQL? Or should we check for `table.schema is not None`?

Reply:
MySQL and Postgre differ on what schemas are.
In postgre : server -> databases -> schemas
In mysql : server -> databases (aka schemas)

You can't* create a table outside a database (ie. outside a schema) in MySQL. If the table has no schema in the ORM model, by default it will be created on the database specified in the connection string (ie. the one in `default_schema_name`).

So if `table.schema` is falsy we use the `default_schema_name`.

It is possible to verify what happens for `''` with this snippet :
```python
t = Table("lake", MetaData(schema=''), autoload_with=conn)
# raises an exception
# (1102, "Incorrect database name ''")
# [SQL: SHOW CREATE TABLE ``.`lake`]
```


*[you can but it involves some specifics](https://dev.mysql.com/doc/refman/8.0/en/innodb-create-table-external.html)

Reply:
Ok, I see. Thanks for the explanations!